### PR TITLE
Add an option for addition label to default node group

### PR DESCRIPTION
**Pull request by Dineshk77**
# Description:
-  Add an option for addition label to default node group

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -221,6 +221,12 @@ variable "default_group_subnet_ids" {
   default     = []
 }
 
+variable "additonal_default_node_group_labels" {
```
Comment (niroz89) [inline variables.tf:224]: Following the same style for the default group extra variables.

```suggestion
variable "default_group_node_labels" {
```

**Review by niroz89 — COMMENTED** [review]
can generate the README.md as well ?

File: node_groups.tf [hunk 2]
```diff
@@ -36,10 +36,10 @@ locals {
       %{endif}
       EOT
 
-    labels = {
+    labels = merge({
       "lifecycle"                                  = "OnDemand"
       "bottlerocket.aws/updater-interface-version" = "2.0.0"
-    }
+    }, var.additonal_default_node_group_labels)
```
Comment (niroz89) [inline node_groups.tf:42]:
```suggestion
    }, var.default_group_node_labels)
```

**Comment by Dineshk77** [comment]
> can generate the README.md as well ?

Added an input section in `README.md` for  `default_group_node_labels`

**Pull request merged**